### PR TITLE
Bug 1887375: Add volumeMode to the create form and yaml of a pvc

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1887375

Screen Shot 2020-10-12 at 10 13 30 PM
Screen Shot 2020-10-12 at 10 13 17 PM

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm

File: frontend/public/components/storage/create-pvc.tsx [hunk 1]
```diff
@@ -250,6 +258,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
...
+      </label>
+      <div className="form-group">
+        {volumeModeRadios.map((radio) => (
+          <RadioInput
+            {...radio}
+            key={radio.value}
+            onChange={handleVolumeMode}
+            inline={true}
+            checked={radio.value === volumeMode}
+            aria-describedby="volume-mode-help"
```
Comment (spadgett) [inline frontend/public/components/storage/create-pvc.tsx:272]: I don't see an element with this ID.

File: frontend/public/components/storage/create-pvc.tsx [hunk 2]
```diff
@@ -250,6 +258,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
...
+      <label className="control-label" htmlFor="volume-mode">
+        Volume Mode
+      </label>
+      <div className="form-group">
+        {volumeModeRadios.map((radio) => (
+          <RadioInput
+            {...radio}
+            key={radio.value}
+            onChange={handleVolumeMode}
+            inline={true}
```
Comment (spadgett) [inline frontend/public/components/storage/create-pvc.tsx:270]: nit: this can just be

```suggestion
            inline
```

**Comment by spadgett** [comment]
/lgtm cancel
since `aria-describedby` looks wrong

File: frontend/public/components/storage/create-pvc.tsx (hunk 1 above)
Comment (jcaianirh) [reply frontend/public/components/storage/create-pvc.tsx:272]: good catch i had one, but removed it ... forgot to take this out.

**Review by spadgett — COMMENTED** [review]
lgtm, do you mind squashing?

**Comment by jcaianirh** [comment]
@spadgett This should be ready to merge

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**